### ISSUE TITLE
VQGAN reconstruction quality

### ISSUE THREAD

**Issue by RoboticsZhang** [joh-fischer/PlantLDM#47]
Hello, thank you for your open-source work, I have gained a lot from it.
When I try to train VQGAN from scratch to reconstruct plant300K data, I find that the reconstructed images are quite blurry, shown as below:
![2023-12-19_00-38](https://github.com/joh-fischer/PlantLDM/assets/61099841/1e033524-8a5b-4116-8fe2-80d016ccb46c)
I didn't change the default hyperparameters (and the default model is VQGANLight, I didn't change it as well), and the loss curves are a bit weird, shown as below:
![image](https://github.com/joh-fischer/PlantLDM/assets/61099841/25881946-7f24-4de0-a824-4f4437575d40)

With this VQGAN pretrained model, I failed to train a good DDPM model.
I wonder that if you can reach a better VQGAN result, and what is the corresponding hyperparameters?

**Comment by RoboticsZhang** [comment]
This is the DDPM trainning results with the above pretrained VQGAN:
![image](https://github.com/joh-fischer/PlantLDM/assets/61099841/0177b67b-6e0d-48d8-9d83-be9104549ee8)
I also trained it with the default hyperparameters.

**Comment by eszaher** [comment]
@RoboticsZhang Any updates on this? Were you able to get high quality reconstructions that preserve high frequencies in the original samples?
